package.json

```diff
@@ -67,12 +67,14 @@
 		"csv": "~5.3.2",
 		"csv-stringify": "~5.6.5",
 		"d3": "~7.8.5",
+		"dompurify": "~3.2.6",
```

Comment:
The update to dompurify and jsdom is not reflected in the package-lock.json file. Since OED does not automatically update package-lock.json on install, it must be done as part of the changes. When I tried to do ``docker compose up`` with this version it failed due to this. This had been done for the previous addition in this PR. Thus, can you carefully update package-lock.json where it only does what is needed for the two updated packages (not a general npm install). Please let me know if you have questions or need help with this.